### PR TITLE
Added onclick  event handler to the buttons

### PR DESCRIPTION
**Pull request by AlierPM**
* Added onclick event handler to buttons
* Added big.js as a dependency
* Added logic folder:
          * calculate.js file added
          * operate.js file added

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,76 +1,117 @@
 import React from 'react';
 import '../App.css';
+import calculate from '../logic/calculate';
```
Comment (fobadara) [inline src/components/Calculator.js:3]: - [OPTIONAL] You don't need to include calculate in the file path. This is enough `../logic`

**Review by fobadara — APPROVED** [review]
Hi @AlierPM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
✅ Descriptive PR
✅ Working calculator

## Optional suggestions
I included some comments here because some files are unavailable for inline comments in this PR.
- [OPTIONAL] I think you should tell us how to clone and use your project in your README.
- [OPTIONAL] Don't forget to add a license file for your MIT link.
- [OPTIONAL] Kindly remember to write your commit messages in the imperative  `Add` instead of `Added`.
- [OPTIONAL] For a project with this amount of changes I would have expected about three to four commits. Please train yourself on this. You might have added one when you included `operate.js` another when you included `calculate.js` and so on.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlierPM** [comment]
Thanks, @fobadara 👏🏿  Your positive review and professional suggestions are noted and appreciated!

**Comment by fobadara** [comment]
You are wecome

**Pull request merged**